renovate.json

```diff
@@ -1,4 +1,4 @@
 {
-	"automerge": false,
+	"automerge": true,
```

Comment:
The test coverage for the project is low (at least in terms of Jest tests) so enabling automerge will just produce more bugs than help us reduce the maintenance of the project.
We can reduce the risk by [only automerging non-major changes](https://docs.renovatebot.com/configuration-options/#automerge) in the deps, but even that is still very dangerous as we can't check if a new bug in a minor version broke a component or not.
I suggest we work on increasing the coverage of the repo first, and once we are happy with it then we can enable auto merges.

Reply:
I don't have strong feelings about this, would leave it up to you @MrSwitch and @ikhemissi to decide how to proceed.

I can also live well entirely without dependency auto-updates. ;)

Reply:
Thanks @ikhemissi, I have now limited auto-updates to non-major changes. There's still a certain risk about minor releases, but updating those deps manually doesn't necessarily make it more likely we'll catch issues with them either. We just need more tests, and those to be more stable, too. At the moment we can't update testing-library/user-events (#195) because this breaks our tests and I can't figure out why. This stuff is the bane of my dev existence. 😅